src/components/nodes/cards/summer-course/summer-course-card.tsx

```diff
@@ -19,9 +20,16 @@ const SummerCourseCard = ({node, headingLevel, ...props}: Props) => {
       imageUrl={node.sumCourseImage?.mediaImage.url}
       imageAlt={node.sumCourseImage?.mediaImage.alt}
     >
-      <Heading className="type-3" id={node.id}>
-        <Link href={node.path}>{node.title}</Link>
-      </Heading>
+      <div className="flex flex-col">
+        <Heading className="type-3 order-2 mb-0" id={node.id}>
+          <ActionLink className="font-roboto font-normal" href={node.path}>
+            {node.title}
+          </ActionLink>
+        </Heading>
+        {node.sumCourseCatalogNumber && (
+          <div className="rs-mb-0 order-1 font-normal">{node.sumCourseCatalogNumber}</div>
```

Comment:
Hmm. I don't see the `node.sumCourseCatalogNumber` being passed in. Do we have a different node data config for teasers? 🤔

Reply:
You have to add the desired fields to the [Graphql query](https://github.com/SU-SWS/summer-nextjs/blob/1.x/src/lib/gql/fragments-node.drupal.gql#L328) and then run `yarn graphql`